Codecademy/gamut
# feat(CareerPathCard): New CareerPathCard

### Overview
This PR adds the existing [CareerPathCard](https://github.com/codecademy-engineering/mono/blob/develop/apps/portal-app/src/components/CatalogPage/CareerPathCard.tsx) component into gamut-labs.

https://github.com/codecademy-engineering/mono/pull/1127 removes it from mono and replaces it with this gamut-labs version.





### PR Checklist

- [ ] Related to JIRA ticket: [DISC-1041]https://codecademy.atlassian.net/browse/DISC-1041

#### Testing Instructions
1. Visit latest styleguide deployment
2. Search for CareerPathCard
3. Ensure that it behaves and is responsive as it is [in production](https://www.codecademy.com/catalog/language/python)

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34440  | [Monolith Env](https://pr-34440-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/1127)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1127)   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |




[DISC-1041]: https://codecademy.atlassian.net/browse/DISC-1041?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
